contents/IFS/code/rust/IFS.rs

```diff
@@ -0,0 +1,36 @@
+use rand::prelude::*;
```

Comment:
Depending on the version of `rand` you are using, you can remove the `::prelude`
```suggestion
use rand::*;
```